lib/bankaccounts/checking-account.rb

```diff
@@ -0,0 +1,48 @@
+module BankAccounts
+
+  # CheckingAccount inherits from Account class
+  class CheckingAccount < Account
+    attr_accessor :checks_used
+
+    def initialize(id, initial_balance)
+      super
+      @min_balance = 0
+      check_initial_balance(initial_balance)
```

Comment:
It looks like I don't need to call it outside of the call within `#super`. I removed it, and all of my specs still pass. Will this break something I'm not seeing?